chore: move crossplane to version stream

kranurag7 opened the pull request:
crossplane is version streamed and this commit migrates crossplane to
the same

kranurag7 committed:
chore: move crossplane to version stream

crossplane is version streamed and this commit migrates crossplane to
the same

developer-guy reviewed (changes requested):
we should add `tag-filter-prefix: 2.1` to updates section.

developer-guy reviewed (dismissed):
lgtm

kranurag7 committed:
add provides for subpackage too

In current state, it's called crossplane-crank and it should be
crossplane-2.1-crank with a provides of crossplane-crank.

kranurag7 committed:
Merge branch 'main' into k7/push-smppmzvnvwvw

kranurag7 committed:
chore: provides crossplane from main package

kranurag7 commented on crossplane-2.1.yaml:
```diff
@@ -1,15 +1,11 @@
 package:
-  name: crossplane
+  name: crossplane-2.1
```
oversight, updated via https://github.com/wolfi-dev/os/pull/73308/commits/53cf1af0f0092fbc39152791e79a0203d7ae474c

AnmolVirdi reviewed (commented):
Minor nit, the rest lgtm.

kranurag7 committed:
chore: remove strip
